packages/fxa-event-broker/lib/version.ts

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*
+ * Return version info based on package.json, the git sha, and source repo
+ *
```

Comment:
Not a full review, moreso a drive-by comment, but did you mean to remove the ./lib/app/version.ts ~copy of this, and fixup the require in ./lib/api/proxy-controller.ts to use this new version.ts?

Reply:
Good point, yes, the original was supposed to be removed in this change!